### ISSUE TITLE
Not working renderLock in iOS 15

### ISSUE THREAD

**Issue by rws08** [gh123man/Refresher#13]
Hi, thanks for the lib.
I found 'renderLock = false' not working after refresh action in iOS 15.
Please check.

```
if canRefresh, !isFingerDown, distance <= 0 {
    renderLock = false
}
```

**Comment by gh123man** [comment]
Sorry about that. Ive reverted the broken change and pushed `1.1.2`. Ill leave this issue open and revisit it once I can fully investigate what is wrong.

**Comment by gh123man** [comment]
Should be all fixed now on `1.1.3` - let me know if you have any other issues.
